src/argilla/client/feedback/dataset/remote/dataset.py

```diff
@@ -480,6 +480,10 @@ def add_records(
         """
         self._records.add(records=records, show_progress=show_progress)
 
+    def search_records(self):
+        # TODO: Call here to argilla.client.sdk.v1.datasets.api.search_records ???
```

Comment:
Yes, and we need to return a list of tuples of [`RemoteFeedbackRecord`, `float`(the schore)]